src/applications/letters/actions/letters.js

```diff
@@ -74,7 +74,11 @@ export function getLetterList(dispatch) {
       } else {
         dispatch({ type: GET_LETTERS_FAILURE });
       }
-      throw new Error(`vets_letters_error_getLetterList: ${status}`);
+      const lettersError = new Error('vets_letters_error_getLetterList');
+      Raven.captureException(lettersError, {
```

Comment:
For consistency with what you've got below, you could even forego defining `lettersError`.